_templates/base.html

```diff
@@ -2,4 +2,13 @@
 {% block extrahead %}
     <script async defer data-domain="napari.org" src="https://plausible.io/js/plausible.js"></script>
     {{ super() }}
+    <style>
+    /* All of this is for the download button which is 
+    declared in download_button.html and included in page: .html
+    
+    This is temporary, and should definitely be replaced with something better
+    when we move to the napari theme.
+    */
+    article{position:relative}.download.icon{color:#000;position:absolute;margin-left:3px;margin-top:12px;width:13px;height:4px;border-radius:1px;border:solid 1px currentColor;border-top:none;top:42px;right:15%}.download.icon:before{content:'';position:absolute;left:6px;top:-9px;width:1px;height:10px;background-color:currentColor}.download.icon:after{content:'';position:absolute;left:4px;top:-4px;width:4px;height:4px;border-top:solid 1px currentColor;border-right:solid 1px currentColor;-webkit-transform:rotate(135deg);transform:rotate(135deg)}.download.icon>span{display:none}.download.icon:hover>span{display:inline;position:absolute;top:1em;left:-3.4em;width:8em;font-size:small;color:#999}
```

Comment:
curious why all the CSS is minimized 😆

Reply:
just cause it's very much not intended to be a long-term editable solution and we're not minimizing after compiling... can undo though if you'd prefer

Reply:
> we're not minimizing after compiling

This is a good point, we'll send less bytes to the browser 😆 

I'm fine with including this then since it won' be a long-term solution.

Reply:
at _least_ 7 bytes! 😂